numpy/core/fromnumeric.py

```diff
@@ -1197,7 +1197,17 @@ def resize(a, new_shape):
     See Also
     --------
     ndarray.resize : resize an array in-place.
-
+    
+    Notes
+    -----
+    Warning: This functionality does **not** consider axes separately,
+    i.e. it does not apply interpolation/extrapolation.
+    It fills the return array with the required number of elements,
+    taken from `a` as they are laid out in memory, disregarding strides and axes.
+    (This is in case the new shape is smaller. For larger, see above.)
+    This functionality is therefore not suitable to resize images,
+    or data where each axis represents a separate and distinct entity.
+    
     Examples
```

Comment:
a final nit: this empty line and one other have spaces, would be nice if you could remove them